src/viser/_gui_api.py

```diff
@@ -867,6 +870,40 @@ def add_button_group(
             )._impl,
         )
 
+    def add_notification(
+        self,
+        title: str,
+        body: str,
+        withCloseButton: bool,
+        loading: bool,
+        autoClose: int | bool = False,
```

Comment:
nit: since this can't be `True` maybe this can be annotated with `int | Literal[false]`? or `int | None` might be more intuitive